docs/ref/unicode.txt

```diff
@@ -272,8 +245,7 @@ querysets are identical::
 Templates
 =========
 
-You can use either strings or UTF-8 bytestrings when creating templates
-manually::
+You can use either text or bytes when creating templates manually::
 
     from django.template import Template
     t1 = Template(b'This is a bytestring template.')
```

Comment:
This doesn't work on master: `TypeError: cannot use a string pattern on a bytes-like object`. That's probably intentional: 3a148f958dddd97c1379081118c30fbede6b6bc4.